charts/ionos-exporter/values.yaml

```diff
@@ -8,10 +8,6 @@ image:
 
 imagePullSecrets: []
 
-nameOverride: ""
-# currently if you do not set the fullnameOverride the too many colons in address error will occur
-fullnameOverride: "ionosexporter"
-
 # credentials
```

Comment:
die vars sollten trotzdem noch in der Datei bleiben, nur leer

Reply:
Ich habe dokumentiert, dass der Default überschrieben werden kann. Reicht doch, oder?